mautic/mautic
# Fix Most hit email redirects dashboard widget

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8424
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR fix https://github.com/mautic/mautic/issues/8424

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Do to the Dashboard
2. Open the widget menu 
![image](https://user-images.githubusercontent.com/49391402/74247345-32ff4280-4ce6-11ea-860d-8e3239edf2c0.png)
3. Add the "Most hit email redirect" widget
4. Set some filter with email that got clicked in it
![image](https://user-images.githubusercontent.com/49391402/74247681-bae54c80-4ce6-11ea-9dd7-06b80630a5f8.png)

5. See that the widget stay blank.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. See your widget works now properly
